galaxy_api/api/v3/viewsets.py

```diff
@@ -94,6 +99,37 @@ def retrieve(self, request, *args, **kwargs):
         response['download_url'] = request.build_absolute_uri(response['download_url'])
         return Response(response)
 
+    @drf_action(
+        methods=["PUT"],
+        detail=True,
+        url_path="certified",
+        permission_classes=api_settings.DEFAULT_PERMISSION_CLASSES + [
+            permissions.IsPartnerEngineer
+        ],
+    )
+    def set_certified(self, request, *args, **kwargs):
+        namespace = self.kwargs['namespace']
```

Comment:
Can you update the request to only require `certification` and not `namespace`, `name` and `version`?

Those values are already provided in the URL and adding them to the request body is redundant.

Reply:
I don't think these values are provided via request body.
`self.kwargs` consists of dictionary of URL path parameters.

Reply:
+1

Reply:
:+1: